components/conforma-verifier-listener/README.md

```diff
@@ -0,0 +1,134 @@
+# Conforma Verifier Listener Component
```

Comment:
Are we happy about this name?

Reply:
It's been updated based on team discussions in another commit.